src/index.ts

```diff
@@ -0,0 +1,17 @@
+import HexColorPicker from "./packages/hex";
+import HexInput from "./packages/HexInput";
```

Comment:
**Discussion: imports and exports**

Shouldn't we migrate to named exports everywhere to make the codebase more consistent?

Bonus: we'll be able to replace these exports with:
```
export { HexColorPicker } from "./components/HexColorPicker";
import { HexColorInput } from "./components/HexColorInput";
```

Reply:
Internally: Could do. Not necessarily a problem to mix default with named though. Personally I usually do default exports with components as find that to be cleaner/simpler, but whatever you think.

For our exports, yeah, would make that much more concise.

Reply:
Done